### PR TITLE
B2B-916 - remove and port SW6 info from SW5 docs repo

### PR DESCRIPTION
**Pull request by dfrancos-hub**

### CODE REVIEW

**Review by Dominik28111 — CHANGES_REQUESTED** [review]

File: products/editions/enterprise-edition/b2b-suite/guides/installation/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+We provide two kinds of virtualization, a vagrant and a docker based solution. Our developers use the docker containers mainly. These containers are also used in our continuous integration process. The supported functions are for both systems equal if the host systems is based on Linux. Instead of testing windows in combination with docker we evaluated the vagrant setup.
+
+If you want to install the B2B-Suite for production environment your system must fit with the defined requirements from the [Shopware core](https://developers.shopware.com/sysadmins-guide/system-requirements/).
+
+### Minimum Requirements
+
+The B2B Suite is based on the minimum requirements of the Shopware core.
+
+#### For Shopware 6
```
[the remark this answers is not in the document]
Comment (dfrancos-hub) [reply products/editions/enterprise-edition/b2b-suite/guides/installation/README.md]: here actually the headline can be removed

File: SUMMARY.md [hunk 2]
```diff
@@ -41,34 +41,35 @@
...
+         * [Storefront](products/editions/enterprise-edition/b2b-suite/guides/storefront/README.md)
+           * [Ajax Panel](products/editions/enterprise-edition/b2b-suite/guides/storefront/ajax-panel.md)
+           * [Product Search](products/editions/enterprise-edition/b2b-suite/guides/storefront/product-search.md)
+           * [Complex views](products/editions/enterprise-edition/b2b-suite/guides/storefront/complex-views.md)
+           * [Modal component](products/editions/enterprise-edition/b2b-suite/guides/storefront/modal-component.md) 
+           * [Company](products/editions/enterprise-edition/b2b-suite/guides/storefront/company.md)
+           * [Acl Routing](products/editions/enterprise-edition/b2b-suite/guides/storefront/acl-routing.md)
+           * [Extending the storefront](products/editions/enterprise-edition/b2b-suite/guides/storefront/how-to-extend-the-storefront.md)
+         * [Administration](products/editions/enterprise-edition/b2b-suite/guides/administration/README.md) 
+         * [Example Plugins](products/editions/enterprise-edition/b2b-suite/guides/example-plugins/README.md)
```
Comment (Dominik28111) [inline SUMMARY.md:72]: please sort it alphabetically.

Comment (dfrancos-hub) [reply SUMMARY.md:72]: the ticket specify to keep the provided structure in the epic

File: SUMMARY.md [hunk 3]
```diff
@@ -41,34 +41,35 @@
...
+           * [Store API](products/editions/enterprise-edition/b2b-suite/guides/core/store-api.md)
+           * [REST API](products/editions/enterprise-edition/b2b-suite/guides/core/rest-api.md)
+           * [Listing service](products/editions/enterprise-edition/b2b-suite/guides/core/listing-service.md)
+           * [CRUD service](products/editions/enterprise-edition/b2b-suite/guides/core/crud-service.md)
+           * [Assignment service](products/editions/enterprise-edition/b2b-suite/guides/core/assignment-service.md)
+           * [Audit Log](products/editions/enterprise-edition/b2b-suite/guides/core/audit-log.md)
+           * [Exception](products/editions/enterprise-edition/b2b-suite/guides/core/exception.md)
+           * [Currency](products/editions/enterprise-edition/b2b-suite/guides/core/currency.md)
+           * [Entity based ACL](products/editions/enterprise-edition/b2b-suite/guides/core/entity-acl.md)
+           * [Line Item List](products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md)
```
[the remark this answers is not in the document]
Comment (Dominik28111) [reply SUMMARY.md:60]: please move it back to concepts as it is a conceptual document.

[the remark this answers is not in the document]
Comment (dfrancos-hub) [reply SUMMARY.md]: ok :)

File: products/editions/enterprise-edition/b2b-suite/guides/installation/README.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+```bash
+./psh.phar mac:init         # build installation
+./psh.phar mac:start        # start apache, mysql 
+./psh.phar mac:stop         # stop apache, mysql
+./psh.phar mac:restart      # restart apache, mysql
+```
+
+You can change the database configuration in your own .psh.yaml file.
+
+```bash
```
Comment (Dominik28111) [inline products/editions/enterprise-edition/b2b-suite/guides/installation/README.md:79]: use yaml here

File: products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md [hunk 5]
```diff
@@ -43,6 +43,6 @@ This basic pattern revolves against other data structures in the component as we
 
 As you can see the specific data is abstracted away through the order context object. 
 An object that can either be generated during the Shopware checkout process or be created dynamically through the API. 
-Here the rule applies: _The B2B-Suite may store or provide ID's, without having an actual concept on what they refer to._
+Here the rule applies: _The B2B-Suite may store or provide ID's, without having an actual concept on what they refer to.
```
Comment (Dominik28111) [inline products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md:46]: image

**Review by Dominik28111 — CHANGES_REQUESTED** [review]

File: products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md (hunk 5 above)
Comment (dfrancos-hub) [reply products/editions/enterprise-edition/b2b-suite/concepts/line-item-list.md]: thanks🙏

**Review by Dominik28111 — APPROVED** [review]

**Review by elkmod — APPROVED** [review]

**Pull request merged**